tests/test_exceptions.py

```diff
@@ -110,11 +110,15 @@ def test_cloud_exception(self):
                     "target": "$search",
                     "message": "$search query option not supported",
                 }
-            ]
+            ],
+            "innererror": {
+                "customKey": "customValue"
+            }
         }
         cloud_exp = self._d(CloudErrorData(), message)
         self.assertEqual(cloud_exp.target, 'query')
         self.assertEqual(cloud_exp.details[0].target, '$search')
+        self.assertEqual(cloud_exp.innererror['customKey'], 'customValue')
```

Comment:
Could you add a `str` call? You're not testing the `__str__` change
```python
self.assertIn("customValue", str(cloud_exp))
```

Reply:
Updated the test. Thanks for the quick review!